### PR TITLE
Fix tree view BC break

### PR DESCRIPTION
**Pull request by Neodork**
Pr for https://github.com/sonata-project/SonataPageBundle/issues/638

### CODE REVIEW

**Comment by greg0ire** [comment]
Hello, and thanks for your contribution!

Please read and apply : https://github.com/sonata-project/SonataAdminBundle/blob/master/CONTRIBUTING.md#the-commit-message

File: Resources/views/PageAdmin/list_tab_menu.html.twig [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {{ knp_menu_render(admin.sidemenu(action), {'currentClass' : 'active'}, 'list') }}
 <p>
     <div class="btn-group">
-        <a type="button" class="btn {% if mode == 'list' %}btn-info active{% else %}btn-default{% endif %}" href="{{ admin.generateUrl('list', { 'filter': { 'site': { 'value': currentSite.id|default('') }}}) }}">
+        <a type="button" class="btn {% if mode == 'list' %}btn-info active{% else %}btn-default{% endif %}" href="{{ admin.generateUrl('list') }}">
             <i class="fa fa-list"></i> {{ 'pages.list_mode'|trans({}, 'SonataPageBundle') }}
         </a>
-        <a type="button" class="btn {% if mode == 'tree' %}btn-info active{% else %}btn-default{% endif %}" href="{{ admin.generateUrl('tree') }}">
+        <a type="button" class="btn {% if mode == 'tree' %}btn-info active{% else %}btn-default{% endif %}" href="{{ admin.generateUrl('tree', { 'filter': { 'site': { 'value': currentSite.id|default('') }}}) }}">
```
Comment (greg0ire) [inline Resources/views/PageAdmin/list_tab_menu.html.twig]: Please linebreak this line.

File: Resources/views/PageAdmin/list_tab_menu.html.twig [hunk 2]
```diff
@@ -1,10 +1,10 @@
 {{ knp_menu_render(admin.sidemenu(action), {'currentClass' : 'active'}, 'list') }}
 <p>
     <div class="btn-group">
-        <a type="button" class="btn {% if mode == 'list' %}btn-info active{% else %}btn-default{% endif %}" href="{{ admin.generateUrl('list', { 'filter': { 'site': { 'value': currentSite.id|default('') }}}) }}">
+        <a type="button" class="btn {% if mode == 'list' %}btn-info active{% else %}btn-default{% endif %}" href="{{ admin.generateUrl('list') }}">
```
Comment (greg0ire) [inline Resources/views/PageAdmin/list_tab_menu.html.twig]: Please linebreak this line.

**Comment by Neodork** [comment]
:+1: Think I got most of it, tell me if I missed some.

**Comment by core23** [comment]
Please have a look at travis and sonata ci @Neodork

**Comment by Neodork** [comment]
@core23 I did, I thought those Composer errors were because of the new versioning is still being applied to this bundle: https://github.com/sonata-project/SonataPageBundle/pull/637

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,12 +1,15 @@
 CHANGELOG
 =========
-
```
Comment (core23) [inline CHANGELOG.md]: Why did you remove this line?

**Comment by Neodork** [comment]
@core23 My bad OCD stuff, didn't even notice.

**Comment by Neodork** [comment]
This is still incomplete I noticed in one of my project. PR on hold till further notice.

**Comment by Soullivaneuh** [comment]
According to the new [Sonata version management](https://github.com/sonata-project/SonataAdminBundle/issues/3053) and next major [release plan](https://github.com/sonata-project/SonataAdminBundle/issues/3731), this project has been refactored regarding branching and versioning.

If you see this message, your PR concerns a patch or a minor release and is not targeting the right branch.

So I'm closing this one, but don't see it as a refusal. If you think your work is still relevant and want to continue, feel free to reopen it on the right branch (e.g. the default one).

Regards.
